src/lib/list/list.scss

```diff
@@ -27,7 +28,18 @@ $mat-dense-three-line-height: 76px;
 $mat-dense-multi-line-padding: 16px;
 $mat-dense-list-icon-size: 20px;
 
-$mat-list-item-inset-divider-offset: 72px;
+// This mixin provides the correct offset for an inset divider based on the
+// size of the parent class (e.g. avatar vs icon)
+@mixin mat-inset-divider-offset($offset) {
+  $mat-list-item-inset-divider-offset: #{(2 * $mat-list-side-padding) + $offset};
+  left: $mat-list-item-inset-divider-offset;
```

Comment:
Why is this one using `left`/`right`, whereas the default divider styles are using `margin-left`/`margin-right`?

Reply:
Changed, the reason was that it was legacy from AngularJS Material